tests/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -114,17 +114,26 @@ class IntegrationSpec extends SpecBase with TestcontainersKafkaLike with Inside
       rebalanceActor2.expectMsg(TopicPartitionsRevoked(subscription2, Set.empty))
       rebalanceActor2.expectMsg(TopicPartitionsAssigned(subscription2, Set(allTps(2), allTps(3))))
 
-      sleep(2.seconds,
+      sleep(4.seconds,
             "to get the second consumer started, otherwise it might miss the first messages because of `latest` offset")
```

Comment:
Unrelated to this PR, but why can't we use a test probe on the second consumer actor to assert when to proceed instead of waiting?